tethys_portal/settings.py

```diff
@@ -360,6 +358,67 @@
 
 ROOT_URLCONF = "tethys_portal.urls"
 
+# Django Tenants settings
+if has_module("django_tenants"):
+    TENANTS_CONFIG = portal_config_settings.pop("TENANTS", {})
```

Comment:
The section in the portal config should include "_CONFIG" to be consistent with the other sections:


```suggestion
    TENANTS_CONFIG = portal_config_settings.pop("TENANTS_CONFIG", {})
```